pypowsybl/network.py

```diff
@@ -94,6 +100,15 @@ def __setstate__(self, state):
         n = _pypowsybl.load_network_from_string('tmp.xiidm', xml, {})
         self._handle = n
 
+    def activate_per_unit(self):
```

Comment:
In order to better separate the code from the base network, and also to be able to manipulate at the same time a per-united view and the "normal" network, I would propose to return a new object here, like a `PerUnitNetwork` which would be a subclass of `Network`:
this way all the per-unit code would go into that class.

Then you could use it this way:
```python
network = pn.load(...)
pu_view = network.per_unit_view(sn=100)
network.get_lines()  # still returns lines not per-united
pu_view.get_lines()  # returns the per-united lines data
```